src/js/tabs/cli.js

```diff
@@ -193,6 +193,21 @@ tab.initialize = function (callback) {
             }
         });
 
+        $('.tab-cli .dumpAll').click(function() {
+            $('.tab-cli .clear').trigger('click');
```

Comment:
Extract the click handler to a named function and call it directly. Then the timeout should not be required.